server.js

```diff
@@ -2,20 +2,62 @@ import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
+import netflixData from './data/netflix-titles.json'
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
+//Mongoose model 
+const Show = mongoose.model("Show", {
+  show_id: {
+    type: Number
+  },
+  title: {
+    type: String
+  },
+  director: {
+    type: String
+  },
+  cast: {
+    type: String
+  },
+  Country: {
+    type: String
+  },
+  release_year: {
+    type: Number
+  },
+  rating: {
+    type: String
+  },
+  duration: {
+    type: String
+  },
+  listed_in: {
+    type: String
+  },
+  description: {
+    type: String
+  },
+  type: {
+    type: String
+  }
```

Comment:
It will be safer for you if you add required: true to some fields. this will prevent from creating empty entries in the database ;)